gokart/in_memory/target.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+from logging import warning
+from typing import Any
+
+from gokart.in_memory.repository import InMemoryCacheRepository
+from gokart.target import TargetOnKart, TaskLockParams
+
+_repository = InMemoryCacheRepository()
+
+
+class InMemoryTarget(TargetOnKart):
+    def __init__(self, data_key: str, task_lock_param: TaskLockParams):
+        if task_lock_param.should_task_lock:
+            warning(f'Redis in {self.__class__.__name__} is not supported now.')
```

Comment:
How about raising exception instead of just showing warning?

```suggestion
            raise ValueError('Redis in {self.__class__.__name__} is not supported now.')
```